src/test/java/com/uid2/shared/store/RotatingSiteStoreTest.java

```diff
@@ -0,0 +1,102 @@
+package com.uid2.shared.store;
+
+import com.uid2.shared.cloud.ICloudStorage;
+import com.uid2.shared.model.ClientSideKeypair;
+import com.uid2.shared.model.Site;
+import com.uid2.shared.store.reader.RotatingClientSideKeypairStore;
+import com.uid2.shared.store.reader.RotatingSiteStore;
+import com.uid2.shared.store.scope.GlobalScope;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Unfortunately we have a mix of JUnit 4 and JUnit 5 tests. Please use JUnit 5 for new tests.

Reply:
changed to junit5